docs/guides/storage.md

```diff
@@ -116,6 +116,9 @@ Note that the original virtual environment is still installed in `$SCRATCH/sqfs-
 
 A benefit of this approach is that the squashfs file can be copied to a location that is not subject to the Scratch cleaning policy.
 
+??? warning
+    Most likely, your virtual environment is not relocatable as it contains symlinks to paths in the original location on the filesystem. Therefore, you need to mount the image in the exact same location where you created the virtual environment.
```

Comment:
```suggestion
    Virtual environment are usually not relocatable as they contains symlinks to absolute locations inside the virtual environment. Therefore, you need to mount the image in the exact same location where you created the virtual environment.
```

Reply:
Is that what you propose? I added an "absolute" relative to your proposal.

Reply:
That's good.
Just one little change remaining: `they contains` -> `they contain`